packages/dds/tree/src/simple-tree/schemaFactory.ts

```diff
@@ -567,11 +567,22 @@ export class SchemaFactory<
 	}
 
 	/**
-	 * Make a field of type identifier instead of the default, which is required.
-	 * @remarks Identifiers may be optionally supplied at node construction time.
-	 * If not supplied, they will be generated automatically when the node is inserted into the tree.
-	 * Attempting to read an automatically generated identifier before the node is inserted into the tree will throw an error.
-	 * An automatically generated identifier will not be present when iterating the nodes's fields until after the node is inserted into the tree.
+	 * A special field which holds a unique identifier for an object node.
+	 * @remarks
+	 * The value of this field, a "node identifier", uniquely identifies a node among all other nodes in the tree.
+	 * Node identifiers are strings, and can therefore be used as lookup keys in maps or written to a database.
+	 * When the node is constructed, the identifier field does not need to be populated.
+	 * The SharedTree will provide an identifier for the node automatically.
+	 * An identifier provided automatically by the SharedTree has the following properties:
+	 * - It is a UUID.
+	 * - It is compressed to a space-efficient representation when stored in the document.
+	 * - A compressed form of the identifier can be accessed at runtime via the `Tree.shortId()` API.
```

Comment:
Nit
```suggestion
	 * - A compressed form of the identifier can be accessed at runtime via {@link Tree.shortId}.
```

Reply:
I've never been able to get links to Tree.* to work, and this is no exception :/ But, also not so bad since Tree is in the top-level namespace so it's easy to find even without a link.